app/controllers/mixins/ems_common_angular.rb

```diff
@@ -59,14 +59,40 @@ def update_ems_button_save
       end
     end
 
-    def update_ems_button_validate(verify_ems = nil)
+    def update_ems_button_validate
+      result, details = realtime_authentication_check
```

Comment:
@jntullo we always call `realtime_authentication_check ` when editing existing provider, wondering if we need to check for session[:selected_roles].try(:include?, 'user_interface') here as well

Reply:
@h-kataria hm, I am not sure if we need it here because the zone isn't considered with `realtime_authentication_check`